### PR TITLE
Add types to docs

### PR DESCRIPTION
**Pull request by isakgustavsen**
There is a description of using types in the vue-sanity docs, but not the nuxt ones. Not sure how you would infer type so i have not added that

### CODE REVIEW

File: docs/content/1.getting-started/3.usage.md [hunk 1]
```diff
@@ -21,11 +21,26 @@ This composable is only available in Nuxt 3. For Bridge, you will need to use `u
 
 ### Example
 
-```ts
+```html [JavaScript]
```
Comment (danielroe) [inline docs/content/1.getting-started/3.usage.md:24]:
```suggestion
```ts
```

**Comment by isakgustavsen** [comment]
Let me know if there is a better way to format this

File: docs/content/1.getting-started/3.usage.md [hunk 2]
```diff
@@ -26,6 +26,21 @@ const query = groq`*[_type == "post" && topic == $topic][0..10]`
...
+::code-group
+```html[Typescript]
+// data will be typed as Ref<Post | null>
+const query = groq`*[_type == "post" && topic == $topic][0..10]`
+const { data, refresh } = useSanityQuery<Post>(query, { topic: 'News' })
+```
+```html[With default values]
+// data will be typed as Ref<string | string> as a string has been provided as a default value
+const query = groq`*[_type == "post" && topic == $topic][0..10]`
+const { data, refresh } = useSanityQuery<Post, string>(query, { topic: 'News' })
```
Comment (danielroe) [inline docs/content/1.getting-started/3.usage.md:41]: This doesn't seem to provide a default value

**Review by danielroe — APPROVED** [review]

**Comment by danielroe** [comment]
Thank you! Feel free to make a follow-up PR if I've omitted anything (like the example of a default value). 🙏

**Pull request merged**